lib/components/map/default-map.tsx

```diff
@@ -341,7 +345,13 @@ class DefaultMap extends Component {
           {/* The default overlays */}
           <EndpointsOverlay />
           <RouteViewerOverlay />
-          <TransitVehicleOverlay ModeIcon={ModeIcon} />
+          <TransitVehicleOverlay
+            id={routeBasedTransitVehicleOverlayNameOverride || undefined}
```

Comment:
maybe move the `|| undefined` into the const so you don't have to repeat it on each of these!